subprojects/build-option/build.gradle.kts

```diff
@@ -7,6 +7,6 @@ gradlebuildJava.usedInWorkers()
 dependencies {
     implementation(project(":cli"))
 
-    implementation(project(":base-annotations"))
+    implementation(project(":base-services"))
```

Comment:
`TextUtil` is declared in `base-services`

Reply:
Can we avoid adding this dependency? `:base-services` is a composite of many things, and if you start depending on it, you get to depend on a lot of things.